src/net/sourceforge/kolmafia/KoLCharacter.java

```diff
@@ -5104,19 +5122,21 @@ public static final Modifiers recalculateAdjustments(
     }
 
     if (HolidayDatabase.getRonaldPhase() == 5) {
-      newModifiers.add(Modifiers.RESTING_MP_PCT, 100, "Moons:Ronald full");
+      newModifiers.addDouble(
+          Modifiers.RESTING_MP_PCT, 100, ModifierType.EVENT, "Moons (Ronald " + "full)");
```

Comment:
pretty sure this string doesn't need splitting

Reply:
fixed.